samples/src/main/python/SynapsePythonSDK.py

```diff
@@ -0,0 +1,17 @@
+# This sample shows how to use the Kusto Python SDK with Synapse authentication.
+# One should use the SDK to run admin commands while using the connector to query
+# https://docs.microsoft.com/azure/data-explorer/kusto/api/python/kusto-python-client-library
+# To upload SDK package to synapse workspace, create a wheel file and follow this:
+# https://docs.microsoft.com/azure/synapse-analytics/spark/apache-spark-manage-python-packages#install-wheel-files
+from azure.kusto.data import KustoClient, KustoConnectionStringBuilder
+
```

Comment:
I would add an example with LS credentials, as that what advance users are looking for